src/python/library/tritonclient/http/_client.py

```diff
@@ -255,6 +265,13 @@ def _post(self, request_uri, request_body, headers, query_params):
         geventhttpclient.response.HTTPSocketPoolResponse
             The response from server.
         """
+        request = Request(headers)
```

Comment:
would it make sense to add in the uri, body, headers, query params into the request?

Reply:
Discussed offline. We're going to hold off on this change until future releases.